packages/frontend/src/utils/urls.ts

```diff
@@ -0,0 +1,2 @@
+export const LogIn = `${process.env.NEXT_PUBLIC_API_URL!}/auth/v1/discord?redirect_uri=${process.env
+	.NEXT_PUBLIC_SITE_URL!}`;
```

Comment:
You should make that `redirect_uri` point to the `/dashboard` page - might have to update API env vars locally for that to work.